packages/projection-typeorm/src/entity/GovernanceAction.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BigIntColumnOptions, OnDeleteCascadeRelationOptions } from './util';
+import { BlockEntity } from './Block.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+
+@Entity()
+export class GovernanceActionEntity {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  // LW-11270
+  // This is required to handle rollbacks, once we have transactions projection
+  // the OnDeleteCascadeRelationOptions can be moved on txId and this column could be removed
+  @ManyToOne(() => BlockEntity, OnDeleteCascadeRelationOptions)
+  @JoinColumn()
+  block?: BlockEntity;
+
+  @Column('varchar')
+  txId?: Cardano.TransactionId;
+
+  @Column('smallint')
+  index?: number;
+
+  @Column('varchar')
+  rewardAccount?: Cardano.RewardAccount;
```

Comment:
We'll probably be storing credentials without bech32 stake address. If that's true, then we should be consistent to be able to reference this in multi-table queries
```suggestion
  stakeCredentialHash?: Hash28ByteBase16;
```

Reply:
I did it, but then I rolled back my change.

The purpose of using credentials is to follow in a more efficient way transactions relevant to a given credential rather than following transactions by addresses.

Here the purpose of this field is not that one, is to describe the governance action... more than this the transaction containing a governance action proposal with a given `rewardAccont` is actually not relevant for it: what will be relevant for that `rewardAccont` will be the _pseudo-reward_ generated by the end of life of the proposal itself.

I think here is more appropriate the original type.

Reply:
> Here the purpose of this field is not that one, is to describe the governance action... more than this the transaction containing a governance action proposal with a given rewardAccont is actually not relevant for it:

Could you please elaborate why it's not relevant? I'm not seeing any upside of this being 'rewardAccount', while credential hash as a benefit of consistency/easier queries. Both are representing the same thing, except that rewardAccount also has a networkId which is irrelevant to store here (db schema is for 1 network only)

Reply:
df1e072db04 should solve this

Reply:
770fd0915c9 should solve this